src/plugins/intel_gpu/src/kernel_selector/kernels/reorder/reorder_kernel_base.cpp

```diff
@@ -189,6 +189,12 @@ ReorderKernelBase::DispatchData ReorderKernelBase::SetDefault(const reorder_para
         dispatchData.lws[0] = 1;
         dispatchData.lws[1] = 16;
         dispatchData.lws[2] = 1;
+    } else if (input_l == DataLayout::byfx) {
+        auto first_primary_axis_size = dispatchData.gws[0];  // X axis
+        auto second_primiary_axis_size =  dispatchData.gws[1];  // YF axes
+        dispatchData.gws[0] = first_primary_axis_size * input.Feature().v;  // takes XF axes
+        dispatchData.gws[1] = second_primiary_axis_size / input.Feature().v;  // takes Y axis
+        dispatchData.lws = {1, 1, 1};
```

Comment:
I think proper place for this code is not `reorder_kernel_base.cpp` but `reorder_kernel.cpp`?

Reply:
Other format handling is here in this function (e.g. input_l == DataLayout::fs_b_yx_fsv32).
 So I moved this code here. Do this condition need to go to reorder_kernel.cpp?